README.md

```diff
@@ -102,6 +102,12 @@ Bitbucket::getDefaultConnection(); // this will return main
 
 // we can change the default connection
 Bitbucket::setDefaultConnection('alternative'); // the default is now alternative
+
+// Get all the repositories info
+Bitbucket::repositories()->list();
+
+// Get all the repositories info filtered by team
+Bitbucket::teams('matteogiacomo')->repositories()->list();
```

Comment:
This team is a bad example because there are no public repos. Are there any teams on Bitbucket that are full of open source software, as a nice example for us to use?

Reply:
Sorry It's a refuse from my script. Can I add it to the config file or as a variable in the script itself?